app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -924,20 +926,19 @@ public void openExternalUrl(Intent intent) {
 
   private void externalLinkPopup(Intent intent) {
     new AlertDialog.Builder(this, dialogStyle())
-        .setTitle(R.string.external_link_popup_dialog_title)
-        .setMessage(R.string.external_link_popup_dialog_message)
-        .setNegativeButton(android.R.string.no, (dialogInterface, i) -> {
-          // do nothing
-        })
-        .setNeutralButton(R.string.do_not_ask_anymore, (dialogInterface, i) -> {
-          sharedPreferenceUtil.putPrefExternalLinkPopup(false);
-          isExternalLinkPopup = false;
-
-          startActivity(intent);
-        })
-        .setPositiveButton(android.R.string.yes, (dialogInterface, i) -> startActivity(intent))
-        .setIcon(android.R.drawable.ic_dialog_alert)
-        .show();
+            .setTitle(R.string.external_link_popup_dialog_title)
```

Comment:
when rebasing it onto master, and formatting the method via ctrl-shift-alt-L, i did not get a change here, same as before? you are sure you want to commit this class? to me it looks like the change is in app/src/main/res/values/styles.xml only.